common/namespace/const.go

```diff
@@ -27,9 +27,18 @@ package namespace
 import "time"
 
 const (
-	// MinRetention is the minimal retention for any namespace (can be
+	// MinRetentionDefault is the minimum retention duration for any namespace (can be
 	// overridden by dynamic config)
-	MinRetention = 1 * 24 * time.Hour
+	MinRetentionDefault = 1 * 24 * time.Hour
```

Comment:
Is this value still used?
If only dynamic config is using this value the plz also remove the dynamic config

Reply:
I thought we still wanted the dynamic config to let administrators enforce a higher limit across all namespaces.

Reply:
there is it used?

validateRetentionDuration seems to only use MinRetentionGlobal and MinRetentionLocal?

https://github.com/temporalio/temporal/pull/1680/files#diff-ffedd6fd51014a5bd6a6c780a0cfecca26bbc1e2e52c769ca34143ab6aa01eaeR55

Reply:
Do you mean where is the dynamic config come in? It's another point in that function, using the `namespaceAttrValidator`. See `handler.go:243` `if err := d.namespaceAttrValidator.validateNamespaceConfig(config)` (and line 523 for Update).

I could put this check in `namespaceAttrValidator` so it's all in one place, but that doesn't seem to have a way to tell if it's a global or local namespace, so I kept it separate. I suppose I could just pass in an `isGlobal` bool to do it there. Maybe that's better.

The other small point is that for Update, I wanted to only do the check if the retention was being changed. Otherwise suppose the dynamic config was changed so that a namespace now has an invalid retention. Any update would fail, even if it's not changing the value, unless it made the retention valid again. It seems fair to only enforce the limit when changing it. But it's a small point and you could argue the other way also.

Reply:
i would propose remove the dynamic config and keep the hard code one, since 1s / 1d for local / global namespace are the min requirement
@yiminc what do you think

Reply:
I like to make it simpler. I'm fine using hardcode value 1d/1s for global / local namespaces. Then you don't have to deal with dynamic config change to a smaller one and now existing ones are invalid. Let's remove the dynamic config for minimum requirement.

Reply:
Sure, I'm always happy to delete code :grin:  
(Please check that I did it right, though)